Jenkinsfile

```diff
@@ -0,0 +1,42 @@
+//
+// Copyright (c) 2020 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+pipeline {
+    options{
+        timestamps()
+        buildDiscarder(logRotator(numToKeepStr: '91'))
+        timeout(time: 12, unit: 'HOURS')
+    }
+    triggers {
+        cron('30 8 * * *')
+    }
+    agent { label 'centos7-docker-4c-2g' }
+    environment {
+        ARCH = 'amd64'
```

Comment:
Just checked and we do have snaps building for ARM as well. We should add a second snap stage and push this down into the stage. Ex: https://github.com/edgexfoundry/edgex-global-pipelines/blob/master/vars/edgeXBuildCApp.groovy#L60-L73